src/mixins/Modal.js

```diff
@@ -2,15 +2,7 @@ import { mapGetters } from 'vuex'
 export default {
   props: ['guid'],
   computed: {
-    ...mapGetters(['getModalState']),
```

Comment:
I still think a Mixin is the best way to have common logic instead of copy-pasting the same code in every modal... thats not easily maintainable. I would prefer improving this mixin instead.